requirements.txt

```diff
@@ -0,0 +1,25 @@
+# Optional for torchaudio.kaldi_io
+numpy
+kaldi_io
+
+# Required for tests only:
+
+# Style-checking for PEP8
+flake8
+
+# Run unit tests
+pytest
+
+# Lets pytest find our code by automatically modifying PYTHONPATH
+pytest-pythonpath
+
+# Coverage statistics
+pytest-cov
+codecov
+
+# Used for comparison of outputs in tests
+librosa
+scipy
+
+# Parallel tests
+pytest-forked
```

Comment:
I'm not a fan of this. Their [README](https://github.com/pytest-dev/pytest-forked) reads

"future maintenance beyond the bare minimum is not planned until a new maintainer is found."

It also says that it doesn't support Windows, but PyTorch has Windows users.